src/app/lib/config/services/index.js

```diff
@@ -1,26 +1,18 @@
 /*
-  This file includes data for the Service Context
-  It will be used by the ServiceContext Component
+  Do not import this file into the primary application.
+  This file is intended for use by tests only where needed. 
+  Including this file in the application will cause the 
+  bundle splitting to stop working.
```

Comment:
This should be homed under `app/testHelpers/` if it's only used in the tests

Reply:
Perhaps, I didn't want to keep them in different parts of the app though

Reply:
Or clarify that this comment is actually saying that this file should not be used/imported directly within the application but should be proxied to via the `import services from '../../lib/config/services/loadableConfig';`

Reply:
I dont follow, nobody should import `loadableConfig` into the app either (other than the 1 place it is)

Reply:
For good housekeeping I've created this [issue](https://github.com/bbc/simorgh/issues/2213) as any non-production code should be homed in a path/have a file name that is explicitly stating it's not for production.